include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -29,11 +29,6 @@ namespace dpl
 {
 namespace unseq_backend
 {
-// helpers to encapsulate void and other types
-template <typename _Tp>
-using void_type = typename ::std::enable_if<::std::is_void<_Tp>::value, _Tp>::type;
-template <typename _Tp>
-using non_void_type = typename ::std::enable_if<!::std::is_void<_Tp>::value, _Tp>::type;
```

Comment:
I'm missing why these are being removed in this PR rather than modified.
Is there a reason to remove them?

Reply:
I suppose these are "implementation details" and would not be likely to be used in user code despite not being "uglified".  OK, I don't have objection here.

Reply:
It was my request. First of all they look ill-designed (API-wise). Second, they are unused. I don't believe it's a user-API. They were not uglified by mistake, I believe.

We can git-blame to make sure.